app/views/comments/_comment_footer.html.erb

```diff
@@ -4,11 +4,10 @@
     id="button-for-comment-<%= comment.id %>"
     data-comment-id="<%= comment.id %>"
     title="heart">
-    <%= inline_svg_tag("small-heart.svg",
-                       aria: true,
-                       class: "crayons-icon reaction-icon not-reacted",
-                       title: t("views.comments.footer.heart.icon")) %>
-    <%= inline_svg_tag("small-heart-filled.svg", aria: true, class: "crayons-icon reaction-icon--like reaction-icon reacted", title: t("views.comments.footer.heart.icon")) %>
+    <%= crayons_icon_tag("small-heart.svg",
```

Comment:
Refactor (potentially blocking): Shouldn't this be `"small-heart"`?

```suggestion
    <%= crayons_icon_tag("small-heart",
```

Reply:
There are two icons right after each other, also in the old code. One is `small-heart`, one is `small-heart-filled`. The `.svg` extension is indeed superfluous but it still works, the helper only appends it if it's not already present 😃 .